device/Microsoft.Azure.Devices.Client/DeviceAuthenticationWithSakRefresh.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Client.Extensions;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    internal class DeviceAuthenticationWithSakRefresh : DeviceAuthenticationWithTokenRefresh
```

Comment:
Since we use Token for the other, maybe calling this Key would make more sense than introducing new acronym.  My 2 cents.

---
In reply to: [150970333](https://github.com/Azure/azure-iot-sdk-csharp/pull/224#discussion_r150970333) [](ancestors = 150970333)